### DISCUSSION TITLE
Want to confirm logic in liquidity range example

### DISCUSSION THREAD

**Discussion by yankev** [atiselsts/uniswap-v3-liquidity-math#12] [Q&A] [answered]
Hi, so on this line: https://github.com/atiselsts/uniswap-v3-liquidity-math/blob/master/subgraph-liquidity-range-example.py#L161
it seems that you're counting liquidity for token1 (token y) when the price is below the lower tick of the current range, however from your document: https://atiselsts.github.io/pdfs/uniswap-v3-liquidity-math.pdf, it seems that the liquidity should all be in token 0 (token x) (scenario 1, on page 2). 
Is there a reason why it's swapped around that I'm missing?

**Comment by yankev** [comment, marked as answer]
oh sorry nvm, I may have confused `tick` for the current tick price.

**Comment by atiselsts** [comment]
I see. In that code `tick` is the loop variable that goes from `min_tick` to `max_tick`, not the current price tick!
